src/AutoRest.CSharp/LowLevel/Output/LowLevelClientMethod.cs

```diff
@@ -27,11 +23,16 @@ public SchemaDocumentation(string schemaName, DocumentationRow[] documentationRo
             }
         }
 
-        public SchemaDocumentation[]? SchemaDocumentations { get; }
-
-        public LowLevelRestClientMethod(string name, string? description, CSharpType? returnType, Request request, Parameter[] parameters, Response[] responses, DataPlaneResponseHeaderGroupType? headerModel, bool bufferResponse, string accessibility, SchemaDocumentation[]? schemaDocumentations) : base (name, description, returnType, request, parameters, responses, headerModel, bufferResponse, accessibility)
+        public LowLevelClientMethod(RestClientMethod restClientMethod, SchemaDocumentation[]? schemaDocumentations, Diagnostic diagnostics)
         {
+            RestClientMethod = restClientMethod;
```

Comment:
Shouln't LowLevelClientMehtod inherit from the RestClientMethod?

Reply:
Ah, it was. Wrapping just feels like a wrong relashionship.

Reply:
I modeled this after the relationship between  `ClientMethod` and `RestClientMethod` that we had for the data-plane stuff.  Can do subclasses if that's preferable.

Reply:
ClientMethod->RestClientMethod  existed because we have the public client with ClientMethod and rest client with RestClientMethods. The LLC only has a single client.